README.md

```diff
@@ -31,8 +31,10 @@ SchemaPlus::Enums is tested on:
 <!-- These lines are auto-generated by schema_dev based on schema_dev.yml -->
 * ruby **2.2.5** with activerecord **4.2**, using **postgresql**
 * ruby **2.2.5** with activerecord **5.0**, using **postgresql**
+* ruby **2.2.5** with activerecord **5.1**, using **postgresql**
```

Comment:
@willsoto we should upgrade the Ruby versions to the latest stable in each branch, following what Rails tests against.